file-formats/sito/type/zif_aff_sito_v1.intf.abap

```diff
@@ -0,0 +1,551 @@
+INTERFACE zif_aff_sito_v1
+  PUBLIC.
+
+  "! <p class="shorttext">SIT2_DO_SCOPE</p>
+  "! SIT2_DO_SCOPE
+  "! $values {@link zif_aff_sito_v1.data:co_sit2_do_scope}
+  TYPES ty_sit2_do_scope TYPE n LENGTH 2.
+
+  CONSTANTS:
+    "! <p class="shorttext">SIT2_DO_SCOPE</p>
+    "! SIT2_DO_SCOPE
+    BEGIN OF co_sit2_do_scope,
+      "! <p class="shorttext">SAP</p>
+      "! SAP
+      sap      TYPE ty_sit2_do_scope VALUE '01',
```

Comment:
Just a minor remark/hint: 
If you prefer SAP in capital letters, you might want to consider the annotation `$enumValue` as described [here](https://github.com/SAP/abap-file-formats/blob/main/docs/json.md#enum-values)

The same might be valid for CDS or V2/V4?

Reply:
Fine for the moment (some effort, too little impact).

Reply:
Perfectly fine for me if you want to keep it as it is. Effort-wise it should be quite simple: 

```suggestion
      "! $enumValue 'SAP'
      sap      TYPE ty_sit2_do_scope VALUE '01',
```